src/main/webapp/jsp/see-all-tasks.jsp

```diff
@@ -0,0 +1,106 @@
+<%@ page import="com.todolist.app.model.Item" %>
+<%@ page import="java.util.Map" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>See all tasks</title>
+    <link rel="stylesheet" href="<%= request.getContextPath() %>/css/see-all-tasks.css">
+    <link href='https://unpkg.com/css.gg@2.0.0/icons/css/more-o.css' rel='stylesheet'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href='https://unpkg.com/css.gg@2.0.0/icons/css/shape-circle.css' rel='stylesheet'>
+</head>
+<body>
+<div class="see-all">
+    <form action="<%= request.getContextPath() %>/see-all-tasks" method="post">
+        <h2>You have <span style="color:#F32276"><%= request.getAttribute("count") %> task(s) </span> to do
```

Comment:
Let's move this style to some class. It's better to avoid `style` let's use css classes